### DISCUSSION TITLE
Sponsorship for development work

### DISCUSSION THREAD

**Discussion by schoeller** [CEXT-Dan/PyRx#155] [Ideas]
Dear all,

as a user of Python in BCAD I would like to be able to sponsor development work through continuous low-level contributions (for long-term application) and punctual contributions for specific functionality. As an example I am able to sponsor Hakan Seven over Patreon and amortize contributions.

Please share your thoughts concerning doability.

Best 

Seb

**Comment by CEXT-Dan** [comment]
I appreciate this enormously! There’s a sponsor feature on GitHub, I haven’t look into it yet.
